modules/template/src/main/twirl/scaladex/view/project/artifacts.scala.html

```diff
@@ -27,7 +27,7 @@
       data-repository="@project.reference.repository">
         <div class="artifacts-filter">
           <div class="result-count">
-            <b>@groupedArtifacts.size</b> @Formats.wordPlural(groupedArtifacts.size, "version") found
+            <b>@groupedArtifacts.values.flatten.map(_._1).size</b> @Formats.wordPlural(groupedArtifacts.values.flatten.map(_._1).size, "artifact") found
```

Comment:
That's good!

A small improvement could be that you don't need to `map(_._1)` to get the size of the collection.

```suggestion
            <b>@groupedArtifacts.values.flatten.size</b> @Formats.wordPlural(groupedArtifacts.values.flatten.size, "artifact") found
```

Also instead of `.flatten.size` you can do `.sum(_.size)` which would be faster when they are a lot of elements, which is not the case in practice.

Reply:
I was extracting only artifact names but I think there is no need, as you mentioned `flatten` does the same and there is no need of `map(_._1)`